src/player.h

```diff
@@ -914,6 +921,7 @@ class player : public Character, public JsonSerializer, public JsonDeserializer
          * the weapon or worn items, If the item is a pointer to an item inside a
          * container, it wont work.
          */
+        // [davek] TODO: Extend this with item `pathname` extension to search nested containers.
```

Comment:
There are two options here, either issue every item a unique index or use a set of tiered indices.

I haven't thought out the implications of both yet.